### PR TITLE
feat: add expired report

### PR DESCRIPTION
**Pull request by dirtyg2120**
* Add `expired` to `Report` model
* Update `expired` to `True` if report exists but not twitter user 
* Generate new report (`expired=False`) if user exist and report is not

Problem: Can not have many report with the same primary key (`twitter_id`), so there are 2 ways:-
1. Composite primary key (`id` and `created_at`)
2. <s>Only update, no save snapshot</s>

### CODE REVIEW

File: app/services/report.py [hunk 1]
```diff
@@ -53,6 +56,7 @@ def add_report(self, twitter_id: str, reporter_id: str) -> Report:
                 scrape_date=user.timestamp,
                 reporters=[reporter_id],
                 score=user.score,
+                expired=True if user is None else False,
```
Comment (phhai-hcmut) [inline app/services/report.py:59]:
```suggestion
                expired=user is None,
```

Comment (dirtyg2120) [reply app/services/report.py:59]: oh :v okay

File: app/tests/test_report.py [hunk 2]
```diff
@@ -111,7 +111,7 @@ def test_two_reports_same_account(
                 if the report is outdated compared to the twitterUser
             - Current: 420 Error User blocked since already report account
         """
-        assert response.status_code == 420
+        assert response.status_code == 404
         pytest.skip("Will be fixed")
```
Comment (produdez) [inline app/tests/test_report.py:115]: Finish the test when ur done

**Review by produdez — CHANGES_REQUESTED** [review]

**Comment by produdez** [comment]
Did you solve the problem mentioned in the PR description? I would go with composite keyl

**Comment by dirtyg2120** [comment]
> Did you solve the problem mentioned in the PR description? I would go with composite keyl

yea i did, the only problem left is currently cant have many records with the same primary key

**Comment by produdez** [comment]
but we would want snapshot instead of overwrite?

**Comment by dirtyg2120** [comment]
i dunno the use of snapshot for now, but i think maybe we will need it later :V. I will go with the composite key

**Comment by produdez** [comment]
We discussed that we want snapshot what are you talking about /(~_~)/

**Comment by dirtyg2120** [comment]
so why u put the "?" in there :V, i thought u ask

**Comment by produdez** [comment]
Cause u seems clueless af and i'm trying to be subtle :v

**Comment by dirtyg2120** [comment]
Done for #31

**Review by produdez — APPROVED** [review]

**Review by phhai-hcmut — APPROVED** [review]

**Pull request merged**